packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -205,6 +222,41 @@ const getSpanElements = (
   return formatInlineElements(typingString, variables);
 };
 
+/**
+ * Helper function to get the string making up the typing indicator string
+ * @param strings TypingIndicatorStrings containing strings to create span elements
+ * @param namesString string of all typing users
+ * @param numTypingUsers number of total typing users
+ * @param numUserNotMentioned number of typing users abbreviated
+ * @returns typing indicator string
+ */
+const getIndicatorString = (
```

Comment:
This would benefit from some unit tests, both to validate it's logic (as it will be v difficult to spot if this ever regresses), and in case someone makes a change they can easily know given some inputs what expected results are

Reply:
will add some in next PR when I will re-introduce the specific aria strings

Reply:
created a task for that: https://skype.visualstudio.com/SPOOL/_workitems/edit/2666070